app/app/Http/Controllers/MemberController.php

```diff
@@ -3,13 +3,14 @@
 namespace App\Http\Controllers;
 
 use App\Member;
+use Illuminate\Support\Carbon;
 
 class MemberController extends Controller
 {
     public function show(Member $member)
     {
         return view('members.show', [
-            'member' => $member,
+            'member' => $member->withGroupMembershipAt(Carbon::now())->find($member->id),
```

Comment:
Beides ein bisschen unschön, muss noch mal drüber nachdenken, was die bessere Lösung wäre.

```suggestion
            'member' => $member,
            'group' => GroupMembership::query()
                ->where('member_id', $member->id)
                ->activeAt(Carbon::now())
                ->first(),
```

oder


```suggestion
            'member' => Member::withGroupMembershipAt(Carbon::now())->find($member->id),
```

Reply:
Ich denke ich fände das zweite schöner.